helmfile/values/falco/falco-common.yaml.gotmpl

```diff
@@ -11,7 +11,10 @@ falco:
 
   grpc_output:
     enabled: true
-
+  config:
+    artifact:
+      install:
+        refs:
```

Comment:
Why does `refs` not have a value here? Is that intentional and if so does that evaluate to an empty list? If you just want an empty list then I'd prefer to explicitly write `refs: []`.
Is this what controls which upstream rules files the we install?